src/AutoRest.CSharp.V3/Output/Models/Client.cs

```diff
@@ -1,31 +1,30 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
+// Licensed under the MIT License.
```

Comment:
I noticed some of these file headers had been changed. Why would we remove the "See License.txt" part?

Reply:
This is the correct header and it got applied to the new files I created.